### DISCUSSION TITLE
add a `getProviders` composable to useAuth

### DISCUSSION THREAD

**Discussion by BayBreezy** [Hebilicious/authjs-nuxt#54] [Ideas]
I was just testing out the module and realized that it does not have a getProviders composable. 

I was in the process of implementing a custom sign in page when I realized this. So on the sign in page, I added this code
```ts
const { data, pending, error, refresh } = await useAsyncData("auth-providers", () =>
  $fetch("/api/auth/providers")
);
```

Not a big deal, just a nice to have

**Comment by Hebilicious** [comment]
That could be a great addition. PR welcome !

**Comment by Hebilicious** [comment]
Implemented
